src/main/java/org/damap/base/conversion/AbstractTemplateExportScienceEuropeComponents.java

```diff
@@ -82,6 +82,9 @@ public void titlePage() {
     if (projectCRIS != null) {
       addReplacement(replacements, "[acronym]", projectCRIS.getAcronym());
       addReplacement(footerMap, "[acronym]", projectCRIS.getAcronym());
+    } else {
+      addReplacement(replacements, "[acronym]", project.getAcronym());
+      addReplacement(footerMap, "[acronym]", project.getAcronym());
```

Comment:
As the acronym is now stored in the database, I think we can remove `projectCris` as well as the condition and just use `project`. If the project was selected from the API before, the acronym should also be set as it was already mapped.

Reply:
Done, the projectCRIS was moved down to be used as parameter for another method